gluster/file/cmd/glusterfile-provisioner/glusterfile-provisioner.go

```diff
@@ -102,14 +102,14 @@ func (p *glusterfileProvisioner) GetAccessModes() []v1.PersistentVolumeAccessMod
 func (p *glusterfileProvisioner) Provision(options controller.VolumeOptions) (*v1.PersistentVolume, error) {
 
 	if options.PVC.Spec.Selector != nil {
-		return nil, fmt.Errorf("claim Selector is not supported")
+		return nil, fmt.Errorf("claim Selectors are not supported")
 	}
 
 	if !util.AccessModesContainedInAll(p.GetAccessModes(), options.PVC.Spec.AccessModes) {
-		return nil, fmt.Errorf("invalid AccessModes %v: only AccessModes %v are supported", options.PVC.Spec.AccessModes, p.GetAccessModes())
+		return nil, fmt.Errorf("invalid AccessModes [%v]: only AccessModes [%v] are supported", options.PVC.Spec.AccessModes, p.GetAccessModes())
 	}
 
-	glog.V(1).Infof("VolumeOptions %v", options)
+	glog.V(1).Infof("VolumeOptions: %v", options)
```

Comment:
not required, for consistency we need to only mention `:` just before error message.

Reply:
It still looks like an improvement to this error message to me, though.